src/python/pants/backend/java/test/junit.py

```diff
@@ -32,40 +29,40 @@ class JavaTestFieldSet(TestFieldSet):
     sources: JavaTestSourceField
 
 
+class JunitTool(JvmToolBase):
```

Comment:
Should this be merged into the Junit subsystem?

Reply:
Yes IMO. We should avoid having multiple subsystems related to the same underlying usecase, as it leaves us with arbitrary suffixes (like `-tool`, `-setup`, `-bootstrap`, etc).

Reply:
Fixed.